### ISSUE TITLE
Enabled new tab settings controls are invisible until you mouse over them

### ISSUE THREAD

**Issue by Smooth-E** [zen-browser/desktop#3037]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Today I noticed the new "Weather" option was added to the new tab page and I wanted to disable it. Upon opening the options side pane I saw that this option has no switch, but when I moused over it, the switch appeared. Though, removing the mouse from this option's box makes the switch disappear again. When the option is turned off, the switch stays visible at all times. Later I discovered that this applies to all options in this pane.

| Mouse over webpage<br>Option enabled | Mouse over the option box<br>Option enabled | Mouse over webpage<br>Option disabled |
| --- | --- | --- |
| ![Switch is invisible](https://github.com/user-attachments/assets/00a4a354-d645-49aa-bfea-13db2663e25a) | ![Enabled switch is visible when moused over](https://github.com/user-attachments/assets/207161cb-7a15-49c6-aa02-5cd6eafee08a) | ![Disabled switch is visible at all times](https://github.com/user-attachments/assets/8dfc155b-082e-4c01-a0f9-9caea2e3c6dd) |

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.19 (Firefox 132.0.1)

### What platform are you seeing the problem on?

Windows

**Comment by rdbo** [comment]
On a side note, shouldn't the weather thing be disabled by default as well?
I just started testing the browser and found that odd

**Comment by Smooth-E** [comment]
> On a side note, shouldn't the weather thing be disabled by default as well? I just started testing the browser and found that odd

I don't think so. When you bring a new feature into your project you either enable it by default if its a quality of life feature, or make an introduction pop-up of some sort, if it is a more focused change, to make those who may need such focused feature aware of it. The weather widget is easy to disable and most people will appreciate additional info on their new tab page, in my opinion

**Comment by rdbo** [comment]
> The weather widget is easy to disable

True, I just brought it up because since privacy seems to be a big thing here, would be weird to ping a third party service by default and show your city location
Also it seems that this weather thing is actually done by Firefox and not Zen (correct me if I'm wrong), so maybe they will consider disabling it in the future

**Comment by Smooth-E** [comment]
> Hi, @Smooth-E. I'm [Dosu](https://dosu.dev), and I'm helping the desktop team manage their backlog. I'm marking this issue as stale.
> 
> **Issue Summary:**
> 
>     * The issue involves invisible toggle switches for new tab settings in the Firefox extension on Windows with version 1.0.1-a.19.
> 
>     * @rdbo raised a concern about the weather widget being enabled by default, suggesting it might conflict with privacy expectations.
> 
>     * You responded that the widget is a quality of life feature and easy to disable, while @rdbo noted potential privacy issues with third-party services.
> 
> 
> **Next Steps:**
> 
>     * Please let me know if this issue is still relevant to the latest version of the desktop repository by commenting here to keep the discussion open.
> 
>     * If there is no further activity, the issue will be automatically closed in 7 days.
> 
> 
> Thank you for your understanding and contribution!

Skipped this comment from the bot. The issue is still present on `1.0.2-b.5`.

@dosubot, consider reopening this issue.

**Comment by Smooth-E** [comment]
> @mr-cheff, the user has indicated that the issue with invisible toggle switches for new tab settings in the Firefox extension is still present on version `1.0.2-b.5`. Could you please take a look and assist them with this?

A small correction to avoid confusion. This is the default Zen's "New Tab" page, and not some kind of extension.
